src/phoenix/trace/semantic_conventions.py

```diff
@@ -35,3 +36,22 @@ class DeploymentAttributes(AttributeGroup):
             type="string",
         ),
     }
+
+
+OUTPUT_VALUE = "output.value"
+OUTPUT_MIME_TYPE = "output.mime_type"
+"""
+The type of output.value. If unspecified, the type is plain text by default.
+If type is JSON, the value is a string representing a JSON object.
+"""
+INPUT_VALUE = "input.value"
+INPUT_MIME_TYPE = "input.mime_type"
+"""
+The type of input.value. If unspecified, the type is plain text by default.
+If type is JSON, the value is a string representing a JSON object.
```

Comment:
Might be nice for these to be slightly grouped together by their namespace.

Reply:
agree, but the code can get complicated for no good reason, so i opted to keeping it as straightforward as possible, emulating [this](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-semantic-conventions/src/opentelemetry/semconv/trace/__init__.py) example